remix.init/netlify-toml

```diff
@@ -2,6 +2,10 @@
 command = "remix build"
 publish = "public"
 
+[dev]
```

Comment:
Required for the Netlify CLI as `remix dev` does not support custom servers.

Reply:
Can we let it run with the built-in server in that case?

Reply:
So the built in server works now with the changes in [netlify/remix-template@`e45fd11` (#95)](https://github.com/netlify/remix-template/pull/95/commits/e45fd11873d0a69e75d611aa21905735b78dfb82)